test/cass.custom.test.js

```diff
@@ -66,8 +68,59 @@ describe('cassandra custom tests', function() {
     m.patNum.should.equal(100);
     m.patStr.should.be.type('string');
     m.patStr.should.equal(cassTestString + '100');
- }
+  }
+
+  describe('create keyspace if it does not exist', function () {
+    it('create keysapce with no specified replication', function (done) {
+      var config = require('rc')('loopback', {
+        test: {
+          cassandra: {
+            host: process.env.CASSANDRA_HOST || 'localhost',
+            port: process.env.CASSANDRA_PORT || 9042,
+            keyspace: 'test0_' + Date.now(),
+            createKeyspace: true,
+          }
+        }
+      }).test.cassandra;
+
+      var ds = new DataSource(require('../'), config);
 
+      cassConnector.initialize(ds, function () {
+        ds.connector.client.execute('SELECT replication FROM system_schema.keyspaces WHERE keyspace_name=\'' + config.keyspace + '\'', function (err, rows) {
```

Comment:
nitpick: could you extract the native query execution to a common function? Since it's also used in the tests afterwards.

Reply:
@jannyHou Thanks for quick review. Appended a change to extract the common code.